subsys/mgmt/mcumgr/smp_shell.c

```diff
@@ -136,10 +140,19 @@ static uint16_t smp_shell_get_mtu(const struct net_buf *nb)
 	return CONFIG_MCUMGR_SMP_SHELL_MTU;
 }
 
+extern struct shell_transport shell_transport_uart;
```

Comment:
You can get rid of this extern to make code a bit cleaner. You are able to access this structure using API function:
`shell_backend_uart_get_ptr()`

Reply:
Done.